src/main/java/reposense/util/FileUtil.java

```diff
@@ -29,26 +28,26 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import com.google.gson.JsonPrimitive;
-import com.google.gson.JsonSerializer;
 
 import reposense.model.CommitHash;
 import reposense.model.FileType;
 import reposense.model.RepoConfiguration;
 import reposense.system.CommandRunner;
 import reposense.system.LogsManager;
+import reposense.util.adapters.DateSerializer;
+import reposense.util.adapters.ZoneSerializer;
 
 /**
  * Contains file processing related functionalities.
  */
 public class FileUtil {
     public static final String REPOS_ADDRESS = "repos";
-
+    public static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
Would it be better to declare this in `DateSerializer.java` since it is only used there?